src/Parser/BrewmasterMonk/Modules/Spells/IronSkinBrew.js

```diff
@@ -17,9 +17,6 @@ class IronSkinBrew extends Module {
   hitsWithoutIronSkinBrew = 0;
   damageWithoutIronSkinBrew = 0;
 
-  // This value is just for dev purposes to check the total, its not shown on the tooltip.
```

Comment:
It doesn't actually need to be there now since i have automated tests, i shouldn't have included it in the tests.